spec/models/recipe_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+
+RSpec.describe Recipe, type: :model do
+  subject do
+    @user = User.create(name: 'Epaltechs', email: 'epal@mail.com', password: 'password')
+    Recipe.create(name: 'Recipe', description: 'This is description', public: true, preparation_time: '1 hours', cooking_time: '1 days', user: @user)
+  end
+
+  before(:each) { subject.save }
+
+  it 'should have name recipe' do
+    expect(subject.name).to eql('Recipe')
+  end
+
+  it 'should have description "This is description"' do
+    expect(subject.description).to eql('This is description')
+  end
+
```

Comment:
- [ ] it would be better to add integrations tests as well  using `capybara`  in order to complete the requirements 
<img width="420" alt="Screenshot 2022-12-16 at 11 07 13" src="https://user-images.githubusercontent.com/61790587/208074829-5c2762dd-9311-47b0-8bef-e052512db452.png">